spec/system/planning_applications/create_spec.rb

```diff
@@ -0,0 +1,101 @@
+# frozen_string_literal: true
```

Comment:
Not sure if this is spec overload, but maybe we should also include a spec to ensure the reviewer can also create an application? And that an invalid user cannot access the form page?

Reply:
on the invalid user, that would be covered by the logins/ local authority specs - since a user belonging to a different local authority cannot login to this one or access anything - and we removed pundit so there were no specific policies added for this use case that would require spec coverage regarding access?


on the reviewer one will it now  :)

Reply:
I don't think we need to duplicate all of them for reviewer, but checking that we don't allow logged out users accessing the form is good

Reply:
I've added it in - although the test spec looks exactly like any "logged out" action so it feels redundant